packit_service/worker/allowlist.py

```diff
@@ -74,66 +67,23 @@ def _strip_protocol_and_add_git(url: Optional[str]) -> Optional[str]:
             return None
         return url.split("://")[1] + ".git"
 
-    def _signed_fpca(self, account_login: str) -> bool:
-        """
-        Check if the user is a packager, by checking if their GitHub
-        username is in the 'packager' group in FAS. Works only the user's
-        username is the same in GitHub and FAS.
-
-        Args:
-            account_login (str): Github username.
-
-        Returns:
-            `True` if user is a packager, `False` otherwise.
-        """
-
-        try:
-            person = self._fas.person_by_username(account_login)
-        except AuthError as e:
-            logger.error(f"FAS authentication failed: {e!r}")
-            return False
-        except FedoraServiceError as e:
-            logger.error(f"FAS query failed: {e!r}")
-            return False
-
-        if not person:
-            logger.info(f"Not a FAS username {account_login!r}.")
-            return False
-
-        for membership in person.get("memberships", []):
-            if membership.get("name") == "cla_fpca":
-                logger.info(f"User {account_login!r} signed FPCA!")
-                return True
-
-        logger.info(f"Cannot verify whether {account_login!r} signed FPCA.")
-        return False
-
-    def add_namespace(self, namespace: str, sender_login: Optional[str] = None) -> bool:
+    @staticmethod
+    def add_namespace(namespace: str) -> bool:
         """
         Add namespace to the allowlist. Namespace is set to `waiting` unless the
         `sender_login` matches packager account in FAS.
```

Comment:
No `sender_login` is passed :) I must've missedthat

Reply:
Because it was used only for checking the Fedora account and added to DB only if there was an automatic match.

Reply:
I meant that it's still mentioned in the comment.

Reply:
aah, sorry :D will remove